css/styles.css

```diff
@@ -0,0 +1,234 @@
+/* pasek nawigacji------------ */
+nav .logo {
+    font-family: 'Pacifico', cursive;
+    font-size: xx-large;
+    margin-left: 8%;
```

Comment:
How about having standarized margins in whole application, that will be stored in a css variable? You could add varaibles like `$margin-sm: 8px`, `$margin-md: 12px`, `$margin-lg: 16px` that you could use freely. Or maybe instead of a margin prefix it could be spacing, so it could be used for paddings also